.devcontainer/devcontainer.json

```diff
@@ -29,18 +29,16 @@
   "customizations": {
     "vscode": {
       "extensions": [
-        "github.vscode-pull-request-github",
-        "eamodio.gitlens",
```

Comment:
I have removed gitlens. Many of its features are not part of vscode itself. Also, it has become quite an annoyance their pop-ups to buy gitkraken.

Users can still opt-in to have it in their devcontainers with `dev.containers.defaultExtensions` setting.